.github/workflows/build.yml

```diff
@@ -92,11 +107,22 @@ jobs:
         # Run one case showing verbose outputs of each library's contents.
         # Run each py file separately as 'pytest -v' (on all files) craps out.
         set -x
-        pytest --capture=tee-sys test_cert_framework_basic.py -k test_getmembers_of_cc_framework
+        pytest --capture=tee-sys test_libcertifier_framework.py -k test_cfslib_getmembers_of_libcertifier_framework
+
+        # Capture output from individual Certifier tests that are run
+        PYTHONUNBUFFERED=TRUE pytest --capture=tee-sys -v test_libcertifier_tests.py
+
+        PYTHONUNBUFFERED=TRUE pytest -v test_libcertifier_framework.py
+
+        PYTHONUNBUFFERED=TRUE pytest -v test_certifier_framework.py
 
-        PYTHONUNBUFFERED=TRUE pytest -v test_cert_framework_basic.py
+        # Just run the basic case of this test to see classes created in generated Python module
+        PYTHONUNBUFFERED=TRUE PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python \
+        pytest --capture=tee-sys -v test_certifier_protobuf_interfaces.py -k test_certifier_pb2_basic
 
-        PYTHONUNBUFFERED=TRUE pytest --capture=tee-sys -v test_certifier_tests.py
+        # Re-run entire test w/o capturing output
+        PYTHONUNBUFFERED=TRUE PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python \
```

Comment:
This `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python` is a protoc-wart we have to run with in order to get these tests to run.